recipe/meta.yaml

```diff
@@ -181,6 +179,157 @@ outputs:
         Compiler for CUDA applications.
       doc_url: https://docs.nvidia.com/cuda/index.html
 
+  - name: cuda-crt-tools
+    files:
+      - bin/crt          # [linux]
+      - Library\bin\crt  # [win]
+    requirements:
+      host:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
+    test:
+      commands:
+        - test -d $PREFIX/bin/crt            # [linux]
+        - test -f $PREFIX/bin/crt/link.stub  # [linux]
+        - if not exist %LIBRARY_BIN%\crt\link.stub exit 1  # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: CUDA internal headers.
+      description: |
+        CUDA internal headers.
```

Comment:
Are these headers? This package is shipping files from `bin/`, not `include/`...

Would "CUDA internal binaries" or "CUDA internal tools" be better?

Reply:
Went with "CUDA internal tools" here https://github.com/conda-forge/cuda-nvcc-impl-feedstock/pull/8/commits/c2b62290f9c5a2dea0c6786dc0cf4b20735d656f